read-fonts/src/font_data.rs

```diff
@@ -232,7 +235,9 @@ impl<'a> Cursor<'a> {
         T: FontReadWithArgs<'a> + ComputeSize,
     {
         let len = len * T::compute_size(args);
-        let temp = self.data.read_with_args(self.pos..self.pos + len, args);
+        let end = self.pos + Saturating(len);
```

Comment:
do we need to explicitly wrap `len` in `Saturating`? just curious because up above in `advance` we seem to be doing arithmetic without the wrapper?

Reply:
I think sometimes it's required or the alternative is to make sure len is typed `u32` explicitly. AFAICS it is usually a question of type inference and only `u32` -> `Saturating<u32>` is inferred.